models/org_team.go

```diff
@@ -34,6 +35,64 @@ type Team struct {
 	Units       []*TeamUnit `xorm:"-"`
 }
 
+// SearchTeamOptions holds the search options
+type SearchTeamOptions struct {
+	UserID      int64
+	UserIsAdmin bool
+	Keyword     string
+	OrgID       int64
+	IncludeDesc bool
+	Limit       int
```

Comment:
I didn't check other API endpoints, but isn't it a bit strange to limit the query while not supporting pagination?

Reply:
Pagination added to the API.